### PR TITLE
Make Sure Gampad Axis Value Is Within Deadzone

### PR DESCRIPTION
**Pull request by zicklag**
The gamepad axis type wasn't making sure the returned value was within the range of the axis thresholds.

### CODE REVIEW

File: src/action_state.rs [hunk 1]
```diff
@@ -522,7 +522,7 @@ pub struct ActionStateDriver<A: Actionlike> {
 ///
 /// This struct is principally used as a field on [`ActionData`],
 /// which itself lives inside an [`ActionState`].
-#[derive(Debug, Clone, PartialEq, Default, Serialize, Deserialize)]
+#[derive(Debug, Clone, PartialEq, Eq, Default, Serialize, Deserialize)]
```
Comment (zicklag) [inline src/action_state.rs:525]: @alice-i-cecile I threw a couple `Eq` derives in here to satisfy clippy, do you know if that's a SemVer breaking change? I don't think it is, but I'm not sure if there's a scenario I'm not thinking of.

Comment (alice-i-cecile) [reply src/action_state.rs:525]: Implementing new traits isn't semver breaking (which is why we can't have negative trait bounds). All good here!

**Comment by zicklag** [comment]
Pushed an extra commit that updates the changelog and bumps the crate version. Let me know if it needs anything else!

**Review by alice-i-cecile — APPROVED** [review]

**Comment by alice-i-cecile** [comment]
All good, thanks!

**Pull request merged**